api/v1/task.proto

```diff
@@ -4,7 +4,7 @@ option go_package = "github.com/tinkerbell/pbnj/api/v1";
 
 package github.com.tinkerbell.pbnj.api.v1;
 
-import "google/protobuf/any.proto";
+//import "google/protobuf/any.proto";
```

Comment:
delete?

Reply:
I'll come back around after this is merged to clean up.

Reply:
updated